headers/functions/arm9.h

```diff
@@ -740,7 +743,9 @@ bool RestoreScriptVariableValues(void* src);
 void InitScenarioScriptVars(void);
 void SetScenarioScriptVar(enum script_var_id id, uint8_t val0, uint8_t val1);
 int GetSpecialEpisodeType(void);
+void SetSpecialEpisodeType(int val);
```

Comment:
Is `enum special_episode_type` applicable for the parameter?

Reply:
Oh, completely missed that, yeah it would